### ISSUE TITLE
将模型切成4份后，第0个进程load错误

### ISSUE THREAD

**Issue by lulu51230** [TsinghuaAI/CPM-Finetune#13]
准备用chid数据集进行微调CPM-large模型，实验使用4块2080ti，且将模型切成4份，微调时报第0个进程无法load的问题。切分模型使用CPM-Generate项目提供的代码。报错详情在训练日志倒数两行。
麻烦大佬们看看这个问题吧。

using world size: 4 and model-parallel size: 4 
 > using dynamic loss scaling
> initializing model parallel with size 4
> initializing model parallel cuda seeds on global rank 0, model parallel rank 0, and data parallel rank 0 with model parallel seed: 26051 and data parallel seed: 23333
100%|███████████████████████████████| 648920/648920 [00:25<00:00, 25446.89it/s]
100%|█████████████████████████████████| 24822/24822 [00:00<00:00, 67924.73it/s]
building GPT2 model ...
 > number of parameters on model parallel rank 1: 651607040
 > number of parameters on model parallel rank 2: 651607040
 > number of parameters on model parallel rank 3: 651607040
130 258
Optimizer = FusedAdam
global rank 1 is loading checkpoint ./CPM-large_MP4/80000/mp_rank_01_model_states.pt
130 258
Optimizer = FusedAdam
global rank 2 is loading checkpoint ./CPM-large_MP4/80000/mp_rank_02_model_states.pt
 > number of parameters on model parallel rank 0: 651607040
130 258
Optimizer = FusedAdam
global rank 3 is loading checkpoint ./CPM-large_MP4/80000/mp_rank_03_model_states.pt
130 258
Optimizer = FusedAdam
learning rate decaying linear
global rank 0 is loading checkpoint ./CPM-large_MP4/80000/mp_rank_00_model_states.pt
A metadata file exists but unable to load model from checkpoint ./CPM-large_MP4/80000/mp_rank_00_model_states.pt, exiting

**Comment by t1101675** [comment]
我们测试了一下，没有发现这个问题，请问您可以提供更多的信息吗？比如 `./CPM-large_MP4/80000/` 的目录结构？

**Comment by lulu51230** [comment]
我试了直接下载的model，也报了这个错，我也把batch调到1,也没有反应。
我在项目readme详情页没有下载到json格式的数据，所以我这边自己处理了一份chid的数据集，数据处理完后没问题，能读入到代码中，或许是模型保存的时候保存了关于数据集的某些信息？

**Comment by t1101675** [comment]
我们直接下载的模型没有保存任何关于数据集的信息。请问您是直接跑的我们提供的脚本吗？是否开启了 deepspeed？因为这个错误看起来是没有开启 deepspeed 才会出现

**Comment by lulu51230** [comment]
我没有开deepspeed，应该是这个问题，感谢您的回答

**Comment by t1101675** [comment]
没事，感谢您的关注！
